cargo-near/src/lib.rs

```diff
@@ -70,6 +76,24 @@ pub struct BuildCommand {
     /// Path to the `Cargo.toml` of the contract to build
     #[clap(long, parse(from_os_str), value_name = "PATH")]
     pub manifest_path: Option<PathBuf>,
+    /// Serialization format to use
+    #[clap(long, value_enum, default_value = "json-min")]
+    pub format: AbiFormat,
+    /// Compression algorithm to use
+    #[clap(long, value_enum, default_value = "zstd")]
+    pub compression: AbiCompression,
```

Comment:
Do we plan to support more than `zstd`? Shouldn't we standardize it so tools that extract ABI can know which algorithm they'd need to decompress the view call response? So maybe this should just be `--uncompressed`?

Reply:
> Do we plan to support more than zstd?

Probably not, but I want to keep our options open just in case. It would be easier to migrate from zstd to snap (hypothetically) if we can have support for both in intermediary versions until we deprecate and delete support for zstd.

Reply:
Agree, if someone really needed to tweak manually themselves, they could just fork this. Better that this is kept as streamlined as possible to make it more easily consumed by tooling